### PR TITLE
[12.0][ADD] hr_attendance_geolocation

### PR DESCRIPTION
**Pull request by AdriaGForgeFlow**
With this module the geolocation of the user is tracked at the check-in/check-out step

cc ~ @Eficent

### CODE REVIEW

**Comment by alexey-pelykh** [comment]
I assume it's WIP?

**Comment by ageficent** [comment]
> I assume it's WIP?

File: hr_attendance_geolocation/data/location_data.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
```
Comment (nikul-serpentcs) [inline hr_attendance_geolocation/data/location_data.xml]:
```suggestion
<odoo noupdate="1">
```

**Comment by AdriaGForgeFlow** [comment]
> I assume it's WIP?

I thinks it's quite complete now. Do you miss something?

**Review by ps-tubtim — APPROVED** [review]
Functional Test Ok. Please fix Codecov.

**Review by JordiBForgeFlow — APPROVED** [review]

**Comment by JordiBForgeFlow** [comment]
@ps-tubtim codecov is ok. Merging now.

**Pull request merged**